### PR TITLE
Bump np versions for build script.

### PR DESCRIPTION
**Pull request by zhangqiaorjc**

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -14,9 +14,9 @@ RUN /pyenv/bin/pyenv install 3.8.0
 RUN /pyenv/bin/pyenv install 3.9.0
 
 # We pin numpy to a version < 1.16 to avoid version compatibility issues.
-RUN eval "$(/pyenv/bin/pyenv init -)" && /pyenv/bin/pyenv local 3.6.8 && pip install numpy==1.15.4 scipy==1.5.4 setuptools wheel six auditwheel
-RUN eval "$(/pyenv/bin/pyenv init -)" && /pyenv/bin/pyenv local 3.7.2 && pip install numpy==1.15.4 scipy==1.5.4 setuptools wheel six auditwheel
-RUN eval "$(/pyenv/bin/pyenv init -)" && /pyenv/bin/pyenv local 3.8.0 && pip install numpy==1.17.3 scipy==1.5.4 setuptools wheel six auditwheel
```
Comment (hawkinsp) [inline build/Dockerfile:19]: Revert the change to Python 3.8.

If 1.17 worked previously there, there's no need to increment to 1.18.

Comment (zhangqiaorjc) [reply build/Dockerfile:19]: done

**Review by hawkinsp — APPROVED** [review]
